draft-ietf-quic-recovery.md

```diff
@@ -1019,7 +1020,10 @@ Since network congestion is not affected by packet number spaces, persistent
 congestion SHOULD consider packets sent across packet number spaces. A sender
 that does not have state for all packet number spaces or an implementation that
 cannot compare send times across packet number spaces MAY use state for just the
-packet number space that was acknowledged.
+packet number space that was acknowledged.  This might result in erroneously
+declaring persistent congestion when including all packet number spaces would
+not have declared persistent connection, but will not lead to a failure to
+detect persistent congestion.
```

Comment:
```suggestion
packet number space that was acknowledged.  This might result in erroneously
declaring persistent congestion, but it will not lead to a failure to detect persistent
congestion. 
```